### PR TITLE
mayhem integration

### PR DESCRIPTION
**Pull request by evanrichter**

### CODE REVIEW

File: fuzz/fuzz_targets/fuzz_target_1.rs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#![no_main]
```
Comment (xansec) [inline fuzz/fuzz_targets/fuzz_target_1.rs:1]: If this is the same as `load_from_str`, can we omit this file?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: fuzz/fuzz_targets/fuzz_target_1.rs (hunk 1 above)
Comment (evanrichter) [reply fuzz/fuzz_targets/fuzz_target_1.rs]: sorry! this was committed by mistake. fixed now

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**